Troubleshooting.md

```diff
@@ -421,38 +436,39 @@ After running `python -m scripts.start`, if you get an error around the followin
          from . import _imaging as core
    ImportError: cannot import name _imaging
 ```
- (note: [Google search results](https://pillow.readthedocs.io/en/stable/installation.html#warnings) indicate that PIL and Pillow cannot coexist in the same environment)
+
+(note: [Google search results](https://pillow.readthedocs.io/en/stable/installation.html#warnings) indicate that PIL and Pillow cannot coexist in the same environment)
```

Comment:
```suggestion
> **Note**  
> [Google search results](https://pillow.readthedocs.io/en/stable/installation.html#warnings) indicate that PIL and Pillow cannot coexist in the same environment
```

it will render like this:
> **Note**  
> [Google search results](https://pillow.readthedocs.io/en/stable/installation.html#warnings) indicate that PIL and Pillow cannot coexist in the same environment

Reply:
Done